src/Playroom/Preview/Iframe.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+
+export default class Iframe extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
Needs hooks 😉

Reply:
Yep, gonna do a sweep of this at some point.